src/components/Snackbar/SnackbarDismissAction.test.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2024 Nordeck IT + Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { act, render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { ComponentType, PropsWithChildren } from 'react';
+import { SnackbarDismissAction } from './SnackbarDismissAction';
+import { SnackbarContextValue, SnackbarProvider } from './SnackbarProvider';
+import { useSnackbar } from './useSnackbar';
+
+describe('<SnackbarDismissAction />', () => {
+  let snackbarContextValue: SnackbarContextValue;
+  let SnackbarProviderTest: ComponentType<PropsWithChildren<{}>>;
```

Comment:
I woulder if we need a special test file for that (looking on other tests), maybe move into SnackbarProvider.test.tsx?

Reply:
In my opinion this is easier to follow: There is a `file` and the test always `file.test` .
No need to look around for the test in other files.